### PR TITLE
create empty db files if they don't exist

### PR DESCRIPTION
**Pull request by kgugala**
this PR is a follow up to #520

### CODE REVIEW

File: utils/environment.sh [hunk 1]
```diff
@@ -44,3 +44,38 @@ if [ $(${XRAY_VIVADO} -h |grep Vivado |cut -d\  -f 2) != "v2017.2" ] ; then
     export XRAY_DIR="/bad/vivado/version"
     return
 fi
+
+# Check if database files exist, if don't create empty ones
```
Comment (litghost) [inline utils/environment.sh]: This doesn't belong in environment.sh.  Maybe a call in ${XRAY_PART}/settings.sh?

Comment (kgugala) [reply utils/environment.sh]: sure, we can move it there but we'd have to call it in every settings file (for that I'd pack this into another script and call the script)
ROI harness minitest introduces it's own settings file. The only common part between all of them is the environment.sh, that is why I put it there.
Shall I add the call every setting file in the project?

Comment (litghost) [reply utils/environment.sh]: enviroment.sh might be common, but it should have side affects.

Have environment.sh define a function is fine, just call it in each settings.sh

Comment (kgugala) [reply utils/environment.sh]: sure, I'll do it that way then

Comment (kgugala) [reply utils/environment.sh]: done

File: experiments/clbpips/generate.sh [hunk 2]
```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
 source ../../utils/environment.sh
+init_db
```
Comment (litghost) [inline experiments/clbpips/generate.sh]: Only call init_db for settings.sh

Comment (kgugala) [reply experiments/clbpips/generate.sh]: fixed

**Pull request merged**